package.json

```diff
@@ -2,6 +2,14 @@
     "name": "@flowforge/flowforge-device-agent",
     "version": "1.9.4",
     "description": "An Edge Agent for running Node-RED instances deployed from the FlowForge Platform",
+    "exports": {
+        "./libraryPlugin": "./lib/plugins/libraryPlugin.js"
+    },
+    "node-red": {
+        "plugins": {
+            "flowforge-library": "lib/plugins/libraryPlugin.js"
```

Comment:
```suggestion
        "./libraryPlugin": "./lib/plugins/node_modules/libraryPlugin.js"
    },
    "node-red": {
        "plugins": {
            "flowforge-library": "lib/plugins/node_modules/libraryPlugin.js"
```

Reply:
I don't think the device agent's package.json should have a node-red section at all - it isn't a module that is intended to provide plugins to anything other than the NR instance it spawns.

Reply:
Agreed - we can add later if that becomes necessary. Will remove